ballerina/tests/47_unary_timestamp_pb.bal

```diff
@@ -105,13 +105,116 @@ public isolated client class UnaryTimestampServiceClient {
         } else {
             message = req;
         }
-        var payload = check self.grpcClient->executeSimpleRPC("UnaryTimestampService/getBiTime", message, headers);
+        var payload = check self.grpcClient->executeSimpleRPC("TimestampService/exchangeTime", message, headers);
         [anydata, map<string|string[]>] [result, respHeaders] = payload;
         return {content: <time:Utc>result.cloneReadOnly(), headers: respHeaders};
     }
+
+    isolated remote function clientStreamTime() returns (ClientStreamTimeStreamingClient|Error) {
```

Comment:
Do we need parentheses here in return type?

Reply:
This is generated from the protobuf tool
@BuddhiWathsala should we change the tool to not generate those parentheses?

Reply:
Ideally, we should. @MadhukaHarith92 @daneshk  WDYT?.

Reply:
yes, @MadhukaHarith92 is parentheses coming along with syntax tree node or we are adding?

Reply:
I think we are adding them manually
https://github.com/ballerina-platform/module-ballerina-grpc/blob/3b433d0ccd4c4b08eecbd77018c8c26e33aacb27/native/src/main/java/io/ballerina/stdlib/grpc/builder/syntaxtree/utils/UnaryUtils.java#L165

Reply:
Let's complete this as a separate task. https://github.com/ballerina-platform/ballerina-standard-library/issues/1739

Reply:
@daneshk, @BuddhiWathsala , yes we have added this parenthesis. I think that was the old Ballerina practice. +1 to remove those parentheses.